src/test/resources/wiremock/mappings/mapping-historicalPriceNoRangeNoDate.json

```diff
@@ -0,0 +1,35 @@
+{
+  "id" : "16369e06-78d4-40a7-98d1-3dbe0ba82fc7",
+  "name" : "tops_hist",
+  "request" : {
+    "url" : "/stock/FB/chart?token=%24%7Bspring.rest.token%7D",
```

Comment:
Look into providing some dummy value during testing, rather than letting it default to just the un-replaced placeholder value.